K8S.md

```diff
@@ -24,12 +24,34 @@ This project deploys an AI pipeline with a multipurpose front end for text gener
 
 Get troubleshoot help on the [FAQ](FAQ.md)
 
+## Requirements
+
+You need to be and administrator.
```

Comment:
"and" should be "an"

Reply:
Changes implemented! Thanks, @peppertech